pkg/job/associator.go

```diff
@@ -45,10 +45,10 @@ func newMetricsToResourceAssociator(dimensionRegexps []*regexp.Regexp, resources
 	return dimensionsFilter
 }
 
-// associateMetricsToResources finds for a cloudwatch.Metrics, the resource that matches the better. If no match is found,
+// AassociateMetricsToResources finds for a cloudwatch.Metrics, the resource that matches the better. If no match is found,
```

Comment:
```suggestion
// AssociateMetricsToResources finds for a cloudwatch.Metrics, the resource that matches the better. If no match is found,
```
Nit: the whole func doc could probably use an update